lms/djangoapps/learner_dashboard/utils.py

```diff
@@ -13,3 +22,34 @@ def strip_course_id(path):
     """
     course_id = unicode(FAKE_COURSE_KEY)
     return path.split(course_id)[0]
+
+
+def disclaimer_incomplete_fields_notification(self, request):
+    """
+    Get the list of fields that are considered as addional but required.
+    If one of these fields are empty, then calculate the numbers of days
+    between the joined date and the current day to decide if display or not
+    the alert after a certain number of days passed from settings or site_configurations.
+    """
+    days_passed = configuration_helpers.get_value('DAYS_PASSED_TO_ALERT_PROFILE_INCOMPLETION', settings.FEATURES.get('DAYS_PASSED_TO_ALERT_PROFILE_INCOMPLETION', 7))
+    user_profile = UserProfile.objects.get(user_id=request.user.id)
+    additional_fields = ['country', 'city', 'gender', 'year_of_birth', 'level_of_education', 'goals']
```

Comment:
For Campus maybe but to make it to upstream this needs to change and not assume everything is required @diegomillan